src/components/screen/Oauth/Oauth.vue

```diff
@@ -80,11 +105,9 @@ export default {
     },
   ],
 
-  data() {
-    return {
-      isLoadingContent: false,
-    };
-  },
+  data: () => ({
+    isLoadingContent: false,
```

Comment:
Just `isLoading` is enough

Reply:
`isLoading` already used for another case